### ISSUE TITLE
bower is not a legal parameter in an Ansible task or handler

### ISSUE THREAD

**Issue by chadfurman** [ansible/ansible-modules-extras#1449]
ansible 1.7.2

```
- name: "Install local ember app dependencies"
  bower: path={{ ember_root_folder }}
```

```
==> default: Running provisioner: ansible...
ERROR: bower is not a legal parameter in an Ansible task or handler
Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

**Comment by sivel** [comment]
The bower module was introduced in Ansible 1.9 as documented within the module and found at http://docs.ansible.com/ansible/bower_module.html

As such, it is not available in 1.7.2.  You will need to upgrade in order to get access to it.

**Comment by chadfurman** [comment]
As a side note, --allow-root using the shell module works, however then all of the bower packages have root privileges while they're installing.
